addons/im_livechat/static/tests/embed/livechat_button.test.js

```diff
@@ -50,8 +50,8 @@ test("open/close persisted channel", async () => {
     await insertText(".o-mail-Composer-input", "How can I help?");
     await triggerHotkey("Enter");
     await assertSteps(["persisted"]);
-    await contains(".o-mail-Message-content", { text: "How can I help?" });
     await waitNotifications([env, "discuss.channel/new_message"]);
+    await contains(".o-mail-Message-content", { text: "How can I help?" });
```

Comment:
Why this should be changed? Order between the 2 await should be irrelevant here.